kafka-rest/src/main/java/io/confluent/kafkarest/ProducerMetricsRegistry.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest;
+
+public class ProducerMetricsRegistry {
+
+  public static final String GROUP_NAME = "produce-api-metrics";
+
+  // sensor names
+  public static final String REQUEST_SENSOR = "request-sensor";
+  public static final String REQUEST_SIZE_SENSOR = "request-size-sensor";
+  public static final String RESPONSE_SENSOR = "response-sensor";
+  public static final String RESPONSE_SIZE_SENSOR = "response-size-sensor";
+  public static final String RECORD_ERROR_SENSOR = "record-error-sensor";
+  public static final String REQUEST_LATENCY_SENSOR = "request-latency-sensor";
+
+  // request
+  public static final String REQUEST_RATE = "request-send-rate";
+  public static final String REQUEST_RATE_DOC = "The average number of requests sent per second.";
+
+  public static final String REQUEST_SIZE_AVG = "request-size-avg";
+  public static final String REQUEST_SIZE_AVG_DOC = "The average request size in bytes.";
+
+  public static final String REQUEST_TOTAL_WINDOWED = "request-total-windowed";
+  public static final String REQUEST_TOTAL_WINDOWED_DOC =
+      "The total number of requests sent within the given window.";
```

Comment:
nit: I'd replace _total_ with _count_ here - the former is mostly used as an adjective to a metric type identifier, while the latter is mostly used as a metric type identifier. Same applies for `RESPONSE_TOTAL_WINDOWED` and `RECORD_ERROR_TOTAL_WINDOWED`.

Reply:
fixed